src/adapter/src/subscribe.rs

```diff
@@ -43,6 +44,8 @@ pub struct ActiveSubscribe {
     pub depends_on: BTreeSet<GlobalId>,
     /// The time when the subscribe was started.
     pub start_time: EpochMillis,
+    /// Are we already in the process of dropping the resources related to this subscribe.
```

Comment:
Let's format this comment to match the other `bool` field `emit_progress`. I.e. "Whether ..."

Reply:
Done!